src/config/dynamic-theme-fixes.config

```diff
@@ -4066,6 +4066,7 @@ vice.com
 
 INVERT
 img[src*="article-logo-news.svg"]
```

Comment:
`img[src*="article-logo-news.svg"]`
`img[src*="article-logo-motherboard.svg"]`
Merge those 2 to `img[src*="article-logo"]`

Reply:
Ok, lol.